osc-server/src/main/java/org/osc/core/broker/util/network/NetworkSettingsApi.java

```diff
@@ -148,4 +90,52 @@ public NetworkSettingsDto getNetworkSettings() {
         return dns;
     }
 
-}
+    private String getNetMask(String ipAddr) {
+        String[] ipAddrParts = ipAddr.split("\\.");
+        String mask = "";
+
+        int octet = Integer.parseInt(ipAddrParts[0]);
+        if (octet <= 127) {
+            mask = "255.0.0.0";
+        } else if (octet >= 128 && octet <= 191) {
+            mask = "255.255.0.0";
+        } else if (octet >= 192 && octet <= 223) {
+            mask = "255.255.255.0";
+        } else if (octet >= 224 && octet <= 239) {
+            mask = "255.0.0.0";
+        } else if (octet >= 240 && octet <= 254) {
+            mask = "255.0.0.0";
+        }
+    return mask;
+    }
+
+    private String getDefaultGateway() {
+        return getNetworkDefaultGateway();
+    }
+
+    private String getNetworkDefaultGateway() {
+
+        String networkField = null;
+        Process process = null;
+        BufferedReader netwokInput = null;
+        try {
+            ProcessBuilder builder = new ProcessBuilder("/bin/sh", NAT_SETTINGS_SCRIPT);
+            process = builder.start();
+            netwokInput = new BufferedReader(new InputStreamReader(process.getInputStream()));
+            String line;
+            while ((line = netwokInput.readLine()) != null) {
+                networkField = line;
+            }
+            int statusCode = process.waitFor();
+            if (statusCode != 0) {
+                log.error("Problem encountered during " + NAT_SETTINGS_SCRIPT + " execution");
+            }
+        } catch (Exception e) {
+            log.error("Exception occured during " + NAT_SETTINGS_SCRIPT + " execution");
+            if (process != null) {
+                process.destroy();
+            }
+        }
+    return networkField;
+    }
+}
```

Comment:
No new line? please check the file format.

Reply:
Did the changes.